src/utilities/util_verify.c

```diff
@@ -61,13 +66,15 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
     size_t size;
     int ch;
     char *config, *dump_offsets, *key, *uri;
-    bool abort_on_error, do_not_clear_txn_id, dump_address, dump_app_data, dump_blocks, dump_layout,
-      dump_pages, read_corrupt, stable_timestamp, strict;
+    bool abort_on_error, do_not_clear_txn_id, dump_address, dump_all_data, dump_key_data,
+      dump_value_data, dump_blocks, dump_layout, dump_pages, read_corrupt, stable_timestamp, strict;
 
-    abort_on_error = do_not_clear_txn_id = dump_address = dump_app_data = dump_blocks =
-      dump_layout = dump_pages = read_corrupt = stable_timestamp = strict = false;
+    abort_on_error = do_not_clear_txn_id = dump_address = dump_all_data = dump_key_data =
+      dump_value_data = dump_blocks = dump_layout = dump_pages = read_corrupt = stable_timestamp =
+        strict = false;
     config = dump_offsets = uri = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "acd:Sstu?")) != EOF)
+
+    while ((ch = __wt_getopt(progname, argc, argv, "acd:Ssktuv?")) != EOF)
```

Comment:
Nit: Seems like it was originally in case-insensitive alphabetical order. To maintain that ordering - `acd:kSstuv`